TSQLLint.Tests/UnitTests/Rules/function-in-predicate/Test.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using TSQLLint.Lib.Rules;
+using TSQLLint.Lib.Rules.RuleViolations;
+
+namespace TSQLLint.Tests.UnitTests.Rules
+{
+    public class FunctionInPredicateTests
+    {
+        private static readonly object[] testCases =
+        {
+            new object[]
+            {
+                "function-in-predicate", "function-in-predicate-where",  typeof(FunctionInPredicate), new List<RuleViolation>()

```

Comment:
I like the always true/false naming convention for the others. Without this its harder to know if this is suppose to pass or fail